src/components/Dog.jsx

```diff
@@ -1,16 +1,33 @@
-import PropTypes from "prop-types";
-import "./Dog.css";
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import './Dog.css';
 
 const Dog = (props) => {
+    const [chipNum, setChipNum] = useState(props.chip);
+
+    //https://www.w3schools.com/js/js_random.asp
+    const getRndInteger = (min, max) => {
+      return Math.floor(Math.random() * (max - min)) + min;
+    };
+  
+    const createChip = () => {
+      setChipNum(getRndInteger(1000, 9999));
+      console.log();
```

Comment:
Was this left over from something? Looks like it can be removed.

Reply:
Oh seems like I overlooked this here. Good eye!